ui/src/views/infra/network/IpRangesTabPublic.vue

```diff
@@ -34,6 +34,18 @@
       :rowKey="record => record.id"
       :pagination="false"
     >
+      <template slot="gateway" slot-scope="record">
+        {{ record.gateway || record.ip6gateway }}
+      </template>
+      <template slot="cidr" slot-scope="record">
+        {{ record.netmask || record.ip6cidr }}
```

Comment:
use `record.cidr` instead of `record.netmask` ?

Reply:
done